components/MobileNav.tsx

```diff
@@ -20,122 +21,132 @@ import {
   NavbarLinkWhyUse
 } from "./NavbarComponents"
 
-const NavBarBoxContainer: FC<
-  React.PropsWithChildren<{ className?: string }>
-> = ({ children, className }) => {
-  return (
-    <div
-      className={`d-flex flex-row align-items-start justify-content-between w-100`}
-    >
-      {children}
-    </div>
-  )
-}
+export const MobileNav: React.FC<React.PropsWithChildren<unknown>> = () => {
+  const BlackCollapse = styled(() => {
+    return (
+      <Navbar.Collapse id="basic-navbar-nav" className="bg-black mt-2 ps-4">
+        {/* while MAPLE is trying to do away with inline styling,   *
+         *  both styled-components and bootstrap classes have been  *
+         *  ignoring height properties for some reason              */}
+        <div style={{ height: "100vh" }}>
+          {whichMenu == "site" ? <SiteLinks /> : <ProfileLinks />}
+        </div>
+      </Navbar.Collapse>
+    )
+  })`
+    .bg-black {
+      background-color: black;
+    }
+  `
 
-const NavBarBox: FC<React.PropsWithChildren<{ className?: string }>> = ({
-  children,
-  className
-}) => {
-  return (
-    <div
-      className={`col d-flex justify-content-start align-items-center ${className}`}
-    >
-      {children}
-    </div>
-  )
-}
+  const ProfileLinks = () => {
+    return (
+      <Nav className="my-4 d-flex align-items-start">
+        <NavbarLinkViewProfile />
+        <NavbarLinkEditProfile
+          handleClick={() => {
+            closeNav()
+          }}
+        />
+        <NavbarLinkSignOut
+          handleClick={() => {
+            closeNav()
+            void signOutAndRedirectToHome()
+          }}
+        />
+      </Nav>
+    )
+  }
+
+  const SiteLinks = () => {
+    return (
+      <Nav className="my-4">
+        <NavbarLinkBills handleClick={closeNav} />
+        <NavbarLinkTestimony handleClick={closeNav} />
+        <NavDropdown className={"navLink-primary"} title={t("about")}>
+          <NavbarLinkGoals handleClick={closeNav} />
+          <NavbarLinkTeam handleClick={closeNav} />
+          <NavbarLinkSupport handleClick={closeNav} />
+          <NavbarLinkFAQ handleClick={closeNav} />
+        </NavDropdown>
+
+        <NavDropdown className={"navLink-primary"} title={t("learn")}>
+          <NavbarLinkEffective handleClick={closeNav} />
+          <NavbarLinkProcess handleClick={closeNav} />
+          <NavbarLinkWhyUse handleClick={closeNav} />
+        </NavDropdown>
+      </Nav>
+    )
+  }
 
-export const MobileNav: React.FC<React.PropsWithChildren<unknown>> = () => {
   const { authenticated } = useAuth()
   const [isExpanded, setIsExpanded] = useState(false)
+  const [whichMenu, setWhichMenu] = useState("site")
   const { t } = useTranslation(["common", "auth"])
 
-  const toggleNav = () => setIsExpanded(!isExpanded)
-  const closeNav = () => setIsExpanded(false)
+  const toggleSite = () => {
+    if (isExpanded && whichMenu == "profile") {
```

Comment:
nit: for these toggles, if we're setting `menu` to the same value in both branches, should we move the `setMenu` call outside the branch?